bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,54 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
-#----alsa install
-sudo apt-get install -y alsa-base alsa-utils
+#----install official requirements
+sudo apt-get install -y alsa-base alsa-utils libasound2-dev
+
+#----install missing pigpio in Raspbian Lite (the command npm install pigpio will be exec by package.json)
+sudo apt-get install pigpio
+
+#----install git and download tjbot
+if [ ! -d "${TJBOT_FOLDER}/recipes/conversation" ]; then
+    sudo apt-get install -y git
+
+    rm -Rf /home/pi/git/tjbot
+    git clone https://github.com/ibmtjbot/tjbot.git /home/pi/git/tjbot
+
+    TJBOT_FOLDER='/home/pi/git/tjbot'
+fi
+
+#----install conversation (to install it will resolve tjbotlib and other dependencies)
+cd $TJBOT_FOLDER
+cd recipes/conversation
+echo "path: $PWD"
+
+sudo npm install --unsafe-perm -g node-gyp
```

Comment:
@fivdi see that attachment. It is from npm install of tests folder but the error is the same for conversation.
[install tests.txt](https://github.com/ibmtjbot/tjbot/files/931296/install.tests.txt)

I'm just thinking a thing... I used a similar command to avoid these errors in tests folder:
sudo npm install --unsafe-perm -g readline-sync

It really install tests because the tests are running fine from bootstrap.sh. So now I'm a little confused with you comment: 
> this command simply installs the node-gyp package globally in /usr/lib/node_modules. It doesn't "install conversation"

If `-g readline-sync` is just installing readline-sync, why tests are running?